demo-app/src/pages/ot/config-builder/index.tsx

```diff
@@ -1,35 +1,6 @@
-import RenderConfig from '@/components/RenderConfig'
-import { useState } from 'react'
-import ComponentList from './ComponentList'
-import ConfigYaml from './ConfigYaml'
-import {
-  CONFIG_BASE,
-  ConfigBuilderContext,
-  INITIAL_FRAMES_PATH,
-} from './config-builder-context'
+import { ConfigBuilderPage } from '@open-truss/open-truss/client'
```

Comment:
I'm loving that we're adopting specialized packages! (the `/client` stuff, dunno what that's called) I think I would suggest we go further though and put all of config builder in `/config-builder` and the playground in `/playground`. They feel optional, and to me that feels like folks should reach into specialized packages to get their contents. What do you think?

Reply:
I think a trade-off of moving pages to top-level imports is we could end up with a lot of top-level imports. Some pages that come to mind that we're building / will build are:
- config builder
- playground
- query editor
- workflow page (tbd)
- workflow session page (tbd)

Maybe instead of `/clients`, this group of features can be `/pages`?

Reply:
Changed `/client` to `/pages`. Let me know if that isn't the route we want to go and I'm happy to revert it https://github.com/open-truss/open-truss/pull/96/commits/e62690b1dff925dc579a7d795c196d5970f1f488

Reply:
That rename makes me feel good!